en.lproj/Localizable.strings

```diff
@@ -3131,6 +3131,134 @@
 "BBQPorkRibs.HowTo.1.Accessibility" = "Enjoy these BBQ pork ribs with salad or rice.";
 
 
+// MARK: Ca Phe Sua (Vietnamese Coffee) Page
+
+/* Vietnamese Coffee Introduction 1 */
+"VietnameseCoffee.Introduction.1" = "##Cà Phê## (coffee) was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese ##Cà Phê## (coffee) special is the used of ##Sữa Đặc## (sweetened condensed milk), because fresh milk was not readily available at the time. The term ##Cà Phê Sữa## is a combination of the ##Cà Phê## (coffee) and ##Sữa Đặc## (sweetened condensed milk) to mean Vietnamese coffee. ##Cà Phê Sữa## (Vietnamese coffee) can be enjoyed hot or cold. The normal presentation of ##Cà Phê Sữa## (Vietnamese coffee) is you have the ##Sữa Đặc## (sweetened condensed milk) in a cup and the ##Phin Lọc## (Phin filter) on top that drips the ##Cà Phê## (coffee) over the ##Sữa Đặc## (sweetened condensed milk).";
+
+/* Vietnamese Coffee Introduction 1 (VoiceOver) */
+"VietnameseCoffee.Introduction.1.Accessibility" = "Coffee was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese coffee special is the used of sweetened condensed milk, because fresh milk was not readily available at the time. Vietnamese coffee can be enjoyed hot or cold. The normal presentation of Vietnamese coffee is you have the sweetened condensed milk in a cup and the Phin filter on top that drips the coffee over the sweetened condensed milk.";
```

Comment:
```suggestion
"VietnameseCoffee.Introduction.1.Accessibility" = "Coffee was introduced to Vietnam in 1857 by a French priest. What makes Vietnamese coffee special is the use of sweetened condensed milk, because fresh milk was not readily available at the time. Vietnamese coffee can be enjoyed hot or cold. Vietnamese coffee is usually presented with the sweetened condensed milk in a cup covered with a Phin filter that drips the coffee over the sweetened condensed milk.";
```